website/content/en/preview/AWS/user-data.md

```diff
@@ -123,3 +125,51 @@ exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
 --kubelet-extra-args '--node-labels=karpenter.sh/capacity-type=on-demand,karpenter.sh/provisioner-name=test  --max-pods=110'
 --//--
 ```
+
+
+## Custom AMIs
+
+You can specify a set of AMIs for a provisioner to use by specifying an AMISelector that identifies AMIs to use through EC2 tags or via a comma-separated list.
+
```

Comment:
```suggestion
```

Reply:
I'm noticing a couple of these. You only need one newline between text and the start of a new section.